src/component/Context.jsx

```diff
@@ -0,0 +1,41 @@
+import { createContext, useEffect, useState } from 'react';
+import { apiPath } from '../api'; 
+
+const MyContext = createContext();
+
+const MyContextProvider = ({ children }) => {
+  const [teams, setTeams] = useState([]);
+  const [members, setMembers] = useState([]);
+  const [timeOff, setTimeOff] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
```

Comment:
You can use Promise.all to fetch data from all endpoints simultaneously, which could help reduce the total loading time. Something like this might work well:
```js
const [teamsRes, membersRes, timeOffRes] = await Promise.all([
          fetch(apiPath('/teams')),
          fetch(apiPath('/members')),
          fetch(apiPath('/timeoff')),
        ]);
```

Reply:
Nice! much better 😎